test/scenarios/README.md

```diff
@@ -116,7 +116,7 @@ installed and pre-configured on the cluster.
 
 # CloudNative - network
 ## Prerequisites
-cilium service mesh
+istio service mesh
```

Comment:
Should we mention in the Readme how to install istio locally with helm?

Reply:
we could link to the official documentation: https://istio.io/latest/docs/setup/install/helm/#installation-steps

Reply:
fine for me, just a hint what to install and what is needed is fine

Reply:
documentation will be improved in a follow up